src/main/java/io/github/moderngamingworlds_mods/woodenutilities/common/util/BiVoidFunc.java

```diff
@@ -0,0 +1,7 @@
+package io.github.moderngamingworlds_mods.woodenutilities.common.util;
+
+@FunctionalInterface
+public interface BiVoidFunc<P1, P2> {
+
+    void apply(P1 param1, P2 param2);
+}
```

Comment:
What about `java.util.function.BiConsumer`? Is that too different?

Reply:
yeah cause you need to return always null in the end and this is not ecstatically good.

```java
public Bla(BiConsumer<Bla, VB, R> la);

new bla((a, b) -> { 
   doSomething();
   return null;
}):

public Bla2(BiVoidFunc<Bla, VB> la);

new Bla2((a, b) -> doSomething()});
```